<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSetup</class>
 <widget class="QWidget" name="WidgetSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WidgetSetup</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>291</width>
     <height>276</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_SRC_LEFT"/>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_DST_WIDTH"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_DST_TOP"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_DST_LEFT"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_SRC_HEIGHT"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_SRC_WIDTH"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_SRC_TOP"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>宽</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>高</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>投影X</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>投影Y</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>投影宽</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="lineEdit_DST_HEIGHT"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>投影高</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="BTN_START">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
